ampersand-form-view.js

```diff
@@ -16,7 +16,11 @@ module.exports = View.extend({
                 var res = {};
                 for (var key in this._fieldViews) {
                     if (this._fieldViews.hasOwnProperty(key)) {
-                        set(res, key, this._fieldViews[key].value);
+                        if (key.match(/\[\]$/)) {
```

Comment:
Could you add a comment above this if statement explaining that you're looking for bracketed field names? I know regex is often hard to grok when reading through code, especially if you haven't been staring at it recently.